contractcourt/contract_resolvers.go

```diff
@@ -822,7 +822,7 @@ func (h *htlcOutgoingContestResolver) Resolve() (ContractResolver, error) {
 	if err != nil {
 		return nil, err
 	}
-	if uint32(currentHeight) >= h.htlcResolution.Expiry {
+	if uint32(currentHeight) >= h.htlcResolution.Expiry-1 {
```

Comment:
We should update the comment to note that we now mine a block earlier and the reason for doing so.